src/components/schemas/expense_policies.yaml

```diff
@@ -0,0 +1,852 @@
+        - If it is set to NOT_IN, this policy rule will be applicable to all the expenses that do not belong to any of the projects listed in the field `expense_project_ids`.
+    expense_cost_center_ids:
+      type: array
+      items:
+        $ref: './fields.yaml#/id_integer'
+      description: |
+        List of cost centers ids considered by this policy rule. <br>
+        _Note: You need to use field `expense_cost_center_ids_op` along with this field._
+      example: [1987, 3456]
+    expense_cost_center_ids_op:
+      allOf:
+        - $ref: ./fields.yaml#/contains_operator
+      description: |
+        - If it is set to IN, this policy rule will be applicable to all the expenses that belong to any of the cost centers listed in the field `expense_cost_center_ids`.
+        - If it is set to NOT_IN, this policy rule will be applicable to all the expenses that do not belong to any of the cost centers listed in the field `expense_cost_center_ids`.
+    expense_allowed_exchange_rate_difference_percentage:
+      type: number
+      description: |
+        Allowed percentage difference for exchange rates beyond which this policy rule will be violated.
+      example: 10
+      minimum: 0
+      maximum: 100
+    expense_allowed_distance_difference_percentage:
+      type: number
+      description: |
+        Allowed distance percentage difference beyond which this policy rule will be violated.<br>
+        _Note: `expense_category_ids` with mileage category is mandatory if the field `expense_allowed_distance_difference_percentage` is present in the request body._
+      example: 8
+      minimum: 0
+      maximum: 100
+    expense_custom_fields:
+      allOf:
+        - $ref: ./fields.yaml#/custom_fields
+      description: |
+        Custom fields (if any)<br>
+        _Example -> "No. of days": 5_ <br>
+        _Note: You need to use field `expense_custom_fields_op` along with this field._
+    expense_custom_fields_op:
+      type: object
+      description: |
+        Operator for the expense_custom_fields. <br>
+        _Format -> field_name: operator_ <br>
+        _Example -> "No. of days": "GREATER_THAN_EQUAL_TO"_ <br><br>
+        Supported values based on field type: <br>
+        `Boolean`: `IS`, `NOT_IS` <br>
+        `Date`: `LESS_THAN_EQUAL_TO`, `GREATER_THAN_EQUAL_TO` <br>
+        `Location`: `IN`, `NOT_IN` <br>
+        `Number`: `LESS_THAN_EQUAL_TO`, `GREATER_THAN_EQUAL_TO` <br>
+        `String`: `CONTAINS`, `EQUALS` <br>
+        `Select`: `IN`, `NOT_IN`<br>
+        _Note: This field is mandatory if the field `expense_custom_fields` is present in the request body._
+      properties:
+        field_name:
+          type: string
+    expense_locations:
+      description: |
+        Location of expense considered by this policy rule.
+        If this is set, then for all such expenses that has location like in case of Flight, Train, Bus, Hotel etc, this policy rule will be applicable. <br>
```

Comment:
, after location

Reply:
If this is set, then for all such expenses that have a location - like expenses of category Flight, Train, Bus, Hotel, etc. - this policy rule will be applicable.